src/components/timeline/TrackContextMenu.js

```diff
@@ -459,7 +459,7 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
       return null;
     }
 
-    const isDisabled = this.getVisibleScreenshotTracks().length === 1;
+    const isDisabled = this.getVisibleScreenshotTracks().length <= 1;
```

Comment:
I'm just curious about why you made this change. It's not wrong so you can leave it, but I'm not sure it was necessary because this code would run only when right clicking on a screenshot track, so there must be one anyway.

Reply:
Ahh! Actually, this is a necessary change.

![ezgif-4-5de4727e81e7](https://user-images.githubusercontent.com/22501334/64884140-937a4d00-d67e-11e9-9e2b-a2b4752b5435.gif)

If we manually hide all screenshots, then the value of ```this.getVisibleScreenshotTracks().length``` becomes 0 and ```hide other screenhots``` option is showed when it makes no sense.

So, it's important to check if it's less than one. Was I able to make that clear?